GenderPayGap.WebUI/Views/AdminOrganisationSector/ViewOrganisationSector.cshtml

```diff
@@ -70,39 +70,65 @@
                     </tr>
                 </thead>
                 <tbody class="govuk-table__body">
-                    @foreach (AuditLog sector in Model.SectorHistory)
+                    @{
+                        List<AuditLog> sectorChanges = Model.SectorHistory;
+                    }
+                    @for (var i = 0; i < sectorChanges.Count; i++)
                     {
-                        var auditLogDetails = JsonConvert.DeserializeObject<AdminChangeSectorAuditLogDetails>(sector.DetailsString);
-                        
+                        AuditLog sectorChange = sectorChanges[i];
+                        var sectorChangeDetails = JsonConvert.DeserializeObject<AdminChangeSectorAuditLogDetails>(sectorChange.DetailsString);
+
                         <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">@(auditLogDetails.OldSector)</td>
-                            <td class="govuk-table__cell">@(auditLogDetails.NewSector)</td>
-                            <td class="govuk-table__cell">@(sector.CreatedDate.ToString("d MMM yyyy"))</td>
+                            @if (i == 0)
+                            {
+                                <th scope="row" class="govuk-table__header">
+                                    Current sector
+                                </th>
+                            }
+                            else if (i == 1)
+                            {
+                                <th scope="row" class="govuk-table__header" rowspan="@(sectorChanges.Count)">
+                                    Previous sectors
+                                </th>
+                            }
+
+                            <td class="govuk-table__cell">@(sectorChangeDetails.NewSector)</td>
+                            <td class="govuk-table__cell">@(sectorChange.CreatedDate.ToString())</td>
                             <td class="govuk-table__cell">
-                                <a href="@Url.Action("ViewUser", "AdminViewUser", new {id = sector.ImpersonatedUser?.UserId ?? sector.OriginalUser.UserId})"
+                                <a href="@Url.Action("ViewUser", "AdminViewUser", new {id = sectorChange.ImpersonatedUser?.UserId ?? sectorChange.OriginalUser.UserId})"
                                    class="govuk-link">
-                                    @(sector.ImpersonatedUser != null ? sector.ImpersonatedUser.Fullname : sector.OriginalUser.Fullname)
+                                    @(sectorChange.ImpersonatedUser != null ? sectorChange.ImpersonatedUser.Fullname : sectorChange.OriginalUser.Fullname)
                                 </a>
                             </td>
-                            <td class="govuk-table__cell">@(auditLogDetails.Reason)</td>
+                            <td class="govuk-table__cell">@(sectorChangeDetails.Reason)</td>
                         </tr>
+
+                        @if (i == sectorChanges.Count - 1)

```

Comment:
This felt kind of messy, but I didn't want to add an AuditLog row if it's _actually_ not from the audit log - this felt like an ok method, but happy to refactor if you think there's a nicer way to handle this!